docs/section5/Rule5-47.md

```diff
@@ -0,0 +1,43 @@
+
+# Envelope - Rule 5-47  
+
+**Rule ID:** 5-47  
+**Rule Description:** The baseline air leakage rate of the building envelope (I75Pa) at a fixed building pressure differential of 0.3 in. of water shall be 1 cfm/ft2.  The air leakage rate of the building envelope shall be converted to appropriate units for the simulation program using one of the methods in Section G3.1.1.4.  
+**Rule Assertion:** Sum of B-RMR infiltration.infiltration.air_leakage_rate = expected value.  
+**Appendix G Section:** Section 5 Envelope  
+**Appendix G Section Reference:** Section G3.1-5(h) Building Envelope Modeling Requirements for the Baseline building  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  None  
+
+**Manual Check:** None  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+**Function Call:** 
+
+  1. get_surface_conditioning_category()
+  2. get_zone_conditioning_category()
+
+## Rule Logic:  
+
+- Get surface conditioning category dictionary for B_RMR: `scc_dict_b = get_surface_conditioning_category(B_RMR)`
+
+- Get zone conditioning category dictionary for B_RMR: `zone_conditioning_category_dict_b = get_zone_conditioning_category(B_RMR)`
+
+- For each zone in the Baseline model: `for zone_b in B_RMR...zones:`
+
+  - For each surface in zone: `for surface_b in zone_b.surfaces:`
+
```

Comment:
should this be for each exterior surface of the zone?

Reply:
The rule requirement (G3.1-5(h)) is "The baseline air leakage rate of the building **envelope** (I75Pa)...". And according to the definition, "**building envelope**: the exterior plus the semiexterior portions of a building." This rule currently checks if the surface is regulated, which includes all exterior (exterior res, exterior non res, exterior mixed) and semi-exterior.

Reply:
The rule requirement (G3.1-5(h)) is "The baseline air leakage rate of the building envelope (I75Pa)...". And according to the definition, "building envelope: the exterior plus the semiexterior portions of a building." This rule currently checks if the surface is regulated, which includes all exterior (exterior res, exterior non res, exterior mixed) and semi-exterior.

Reply:
I agree with @xingcx